substrate/frame/revive/src/benchmarking/mod.rs

```diff
@@ -558,6 +558,33 @@ mod benchmarks {
 		assert_eq!(result.unwrap(), 1);
 	}
 
+	#[benchmark(pov_mode = Measured)]
+	fn seal_to_account_id() {
+		// use a mapped address for the benchmark, to ensure that we bench the worst
+		// case (and not the fallback case).
+		let caller = whitelisted_caller();
+		let origin: T::RuntimeOrigin = RawOrigin::Signed(caller.clone()).into();
+		Contracts::<T>::map_account(origin.clone()).unwrap();
+		let address = T::AddressMapper::to_address(&caller);
```

Comment:
Should also fix the type inference error (hopefully).

```suggestion
        let address = {
                let caller = account("seal_to_account_id", 0, 0));
                T::AddressMapper::map(&caller).unwrap();
                T::AddressMapper::to_address(&caller)
        };
```

Reply:
Just applied your suggestion. I didn't get a type inference error locally though when running the benchmark, where did you see it?

Reply:
The CI failed with the error.